projectatomic/ai-studio
# feat: allow to swap model

### What does this PR do?

This PR allows to swap models and start the application with the selected one.
Need to add some tests but it works if you want to test. On Windows the mistral model is faster than llama

### Screenshot / video of UI

![swap_model](https://github.com/projectatomic/ai-studio/assets/49404737/6c2d36f3-9d14-4756-96d4-02d8c34a645a)

### What issues does this PR fix or reference?

it resolves #201 

### How to test this PR?

1. select the chatbot recipe, swap model, run application and check it uses the selected model
